octokit-project/src/main/java/org/example/calculator/Calculator.java

```diff
@@ -0,0 +1,15 @@
+package org.example.calculator;
+
+public class Calculator {
+
+    public static void main(String[] args) {
+        int a = 1;
+        int b = 2;
+
```

Comment:
[Severity: Minor] The addition operation is performed directly in the main method without encapsulating it in a separate method.
Consider creating a method for the addition operation (e.g., `public static int add(int x, int y)`).
Rationale: This adheres to the principle of separation of concerns, making the code more modular and easier to test or extend in the future.
Reference: SOLID principles, specifically the Single Responsibility Principle.